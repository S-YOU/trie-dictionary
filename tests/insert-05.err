nodes: 2
nodes: 3
nodes: 4
nodes: 5
nodes: 6
