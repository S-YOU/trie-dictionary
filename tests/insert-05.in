insert iiqxg
insert iiqxg
insert iiqx
insert iiqxgpyyxp
insert iiqxgp
insert ii
insert iiqxgp
insert iiqxgpyyxp
insert ii
insert ii
