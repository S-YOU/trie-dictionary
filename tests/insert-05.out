word number: 0
ignored
word number: 1
word number: 2
word number: 3
word number: 4
ignored
ignored
ignored
ignored
